customer-service/src/main/java/com/epam/customerservice/dto/GoodsType.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.customerservice.dto;
+
+public enum GoodsType {
```

Comment:
Not an issue for an educational project, but in production an enum would be a constraint. What if you need to add a type later? I recommend just a String.

Reply:
Got it!